packages/api/src/models/defaultAnimalTypeModel.js

```diff
@@ -39,6 +40,30 @@ class DefaultAnimalTypeModel extends Model {
       additionalProperties: false,
     };
   }
+
```

Comment:
Oh gosh, the query is completely identical except for the table + column name? I think we should declare our experimental two-table approach a failure 😂

I like having this as a codebase example of how to do SQL in a model though. Even for an SQL-phobe like me, I think it reads very clearly side by side with the model definition ❤️

Reply:
For the custom type, we also need to check `farm_id`, but we could do `farm_id = ? OR farm_id is NULL` if we had one table.